test/integration/package-react/package.json

```diff
@@ -16,8 +16,8 @@
     }
   },
   "dependencies": {
-    "react": "latest",
-    "react-dom": "latest"
+    "react": "16.13.1",
+    "react-dom": "16.13.1"
```

Comment:
Unfortunately Yarn wanted this to be version-locked. It was throwing an error in our Windows tests.

This might be fine—this is the latest version as of now, React doesn’t release often, and this is easy to update for our tests.